mne_bids/tests/test_write.py

```diff
@@ -564,6 +664,11 @@ def test_set(_bids_validate):
     write_raw_bids(raw, bids_basename, output_path)
     _bids_validate(output_path)
 
+    # test anonymize and convert
+    raw.info['meas_date'] = (np.int32(1038942070), np.int32(0))
```

Comment:
same question, what is this strange number and why?

Reply:
I'm unresolving this because anonymize should not fail with `raw.info['meas_date']`. Can you check if it works on the PR of @bloyl